README.md

```diff
@@ -18,6 +18,8 @@ Role Variables
 |--------------------------------------------|-----------------------|-------------------------------------------|
 | ovirt_repositories_ovirt_release_rpm       | UNDEF                 | URL of oVirt release package, which contains required repositories configuration. |
 | ovirt_repositories_use_subscription_manager| False                 | If true it will use repos from subscription manager and the value of ovirt_repositories_ovirt_release_rpm will be ignored |
+| ovirt_repositories_ovirt_version           | 4.2                   | oVirt release version. Will be used to enable the required repositories in case ovirt_repositories_use_subscription_manager is set. |
```

Comment:
I think we should mention possible values here: `master, 4.2,  4.1` Is that correct? But for subscription manager use case we need eliminate master, so we should probably document that

Reply:
Sure, this variable is used only for 4.1 and 4.2 BETA now. So I will add that supported options are [4.1, 4.2], ok?

Reply:
Ahh OK, I forgot that we specify ovirt_repositories_ovirt_release_rpm directly. So yes, for now let's limit to 4.1 and 4.2, in future we could use this version also for release rpm if release rpm is not specified.

Thanks